deepchem/models/dft/dftxc.py

```diff
@@ -74,6 +74,13 @@ def forward(self, inputs):
                 qcs.append(evl.run(system))
             if entry.entry_type == 'dm':
                 output.append((torch.as_tensor(entry.get_val(qcs)[0])))
+            elif entry.entry_type == 'dens':
```

Comment:
We should document this calculation in the docstring. We also need to explain we don't support predictions for density profiles at present

Reply:
We should provide the needed outputs for the loss computation but perform the actual loss in the XCLoss. Have outputs
```
val, volume
```
from the forward pass for density formulas